wifi: SAP mode docs (#13562)
nrfconnect/sdk-nrf: doc/nrf/device_guides/working_with_nrf/nrf70/features.rst

Supported Wi-Fi standards and modes
***********************************

The nRF70 Series wireless companion ICs add Wi-Fi 6 support to a host device that includes IP-based networking support.
Wi-Fi 6 aligns with IEEE 802.11ax and all earlier versions of the IEEE 802.11 suite of wireless LAN standards.

> Currently :abbr:`Station (STA)` and ::abbr:`Software-enabled Access Point (SoftAP or AP)` modes support is included.

Reviewer: IMO, using :term: is more relevant than :abbr:.
```suggestion
Currently, :term:`Station (STA)` and :term:`Software-enabled Access Point (SoftAP or SAP)` modes are supported.
```

> Currently, :term:`Station mode (STA)` and :term:`Software-enabled Access Point (SoftAP or SAP)` modes are supported.

Reviewer: ```suggestion
Currently, :term:`Station mode (STA)` and :term:`Software-enabled Access Point (SoftAP or SAP)` mode are supported.
```

The nRF70 series wireless companion ICs add Wi-Fi 6 support to a host device that includes IP-based networking support.
Wi-Fi 6 aligns with IEEE 802.11ax and all earlier versions of the IEEE 802.11 suite of wireless LAN standards.

> Currently Station (STA) and Soft Access Point (AP) modes support is included.

Author: I think we should just use the acronym that was introduced in the `glossary`.
